domain/camera/src/main/java/com/google/jetpackcamera/domain/camera/CameraXCameraUseCase.kt

```diff
@@ -289,8 +319,12 @@ constructor(
             } ?: false
 
         val previewUseCaseBuilder = Preview.Builder()
-        if (isPreviewStabilizationSupported) {
-            previewUseCaseBuilder.setPreviewStabilizationEnabled(true)
+        if (isPreviewStabilizationSupported && shouldStabilizePreview != Stabilization.UNDEFINED) {
+            val isStabilized = when (shouldStabilizePreview) {
+                Stabilization.ON -> true
+                else -> false
```

Comment:
nit: use Stabilization.OFF branch instead of else. We don't need an else because it's an enum